### PR TITLE
Fix ApiDocTest unexpected documented exported package issue

### PR DESCRIPTION
**Pull request by akurtakov**
Use wiring api instead of the old PlatformAdmin.

### CODE REVIEW

File: org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java [hunk 1]
```diff
@@ -276,31 +277,17 @@ private static InputSource getExtensionPointSchemaSource(IExtensionPoint extensi
...
-//					if (child.endsWith(".class")) {
-//						exportedPackageIds.add(name);
-//						break pack;
-//					}
-//				}
-//				System.out.append("Exported package without class files: ").append(name).append('\n');
-			}
+		for (BundleDescription bundle : state.getBundles()) {
+			bundle.getCapabilities(BundleRevision.PACKAGE_NAMESPACE)
+					.forEach((t) -> exportedPackageIds.add((String) t.getAttributes().get("osgi.wiring.package")));
```
Comment (mickaelistria) [inline org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java:290]: "osgi.wiring.package" is actually the value of BundleRevision.PACKAGE_NAMESPACE.

File: org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java [hunk 2]
```diff
@@ -276,31 +277,17 @@ private static InputSource getExtensionPointSchemaSource(IExtensionPoint extensi
...
 		BundleContext context = FrameworkUtil.getBundle(ApiDocTest.class).getBundleContext();
 		ServiceReference<PlatformAdmin> platformAdminReference = context.getServiceReference(PlatformAdmin.class);
 		PlatformAdmin service = context.getService(platformAdminReference);
 		State state = service.getState(false);
-		ExportPackageDescription[] exportedPackages = state.getExportedPackages();
-		for (ExportPackageDescription exportPackageDescription : exportedPackages) {
-			String name = exportPackageDescription.getName();
-			if (!NON_API_PACKAGES.matcher(name).matches()) {
-				if (Boolean.TRUE.equals(exportPackageDescription.getDirective("x-internal"))
-						|| exportPackageDescription.getDirective("x-friends") != null) {
```
Comment (mickaelistria) [inline org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java:290]: So we check for all docs now, including internal/friends/NON_API_PACKAGES ?
I don't think it's too bad, just want to make sure it's the actual intent/fix.

Comment (akurtakov) [reply org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java:290]: Actually it is the opposite. These package names are removed from packageIds and it's checked whether there is anything left. IMHO, it's fine to have x-internal/x-friend packages in the javadoc and this test shouldn't fail in such a case.

File: org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java (hunk 1 above)
Comment (akurtakov) [reply org.eclipse.ua.tests.doc/src/org/eclipse/ua/tests/doc/internal/linkchecker/ApiDocTest.java:290]: Thanks, will try to remember it for the next time I play with wiring.

**Pull request merged**